x-pack/legacy/plugins/ml/public/data_frame/common/job.ts

```diff
@@ -39,3 +42,35 @@ export interface DataFrameTransformWithId extends DataFrameTransform {
 
 // Don't allow intervals of '0', don't allow floating intervals.
 export const delayFormatRegex = /^[1-9][0-9]*(nanos|micros|ms|s|m|h|d)$/;
+
+export enum REFRESH_TRANSFORM_LIST_STATE {
+  ERROR = 'error',
+  IDLE = 'idle',
+  LOADING = 'loading',
+  REFRESH = 'refresh',
+}
+export const refreshTransformList$ = new BehaviorSubject<REFRESH_TRANSFORM_LIST_STATE>(
+  REFRESH_TRANSFORM_LIST_STATE.IDLE
+);
+
+export const useRefreshTransFormList = () => {
```

Comment:
Nit: Everywhere else `Transform` is not camel cased like `TransForm` - maybe `useRefreshTransformList` should be what this is called for consistency?

Reply:
of course, totally missed that, thanks!!

Reply:
Fixed in bd1ea44.